### ISSUE TITLE
Stoke on trent city council 

### ISSUE THREAD

**Issue by Doghousestaffs** [mampfes/hacs_waste_collection_schedule#1148]
Hi, 

could you please add stoke on trent city council to your integration.

tbanks.

**Comment by bbr111** [comment]
Can you provide 1 or 2 sample addresses? best with all available bin types.

This link with the correct UPRN delivers good and parseable informations.
https://www.stoke.gov.uk/jadu/custom/webserviceLookUps/BarTecWebServices_missed_bin_calendar.php?UPRN=???

**Comment by Doghousestaffs** [comment]
> Can you provide 1 or 2 sample addresses? best with all available bin types.
> 
> This link with the correct UPRN delivers good and parseable informations. https://www.stoke.gov.uk/jadu/custom/webserviceLookUps/BarTecWebServices_missed_bin_calendar.php?UPRN=???

Thanks for responding

try 8 Bleriot close St37qr 
And 76 Broadway st35lg

**Comment by Doghousestaffs** [comment]
@bbr111 had much luck ? Think the uprn for mine is 203455011390 

thanks

**Comment by bbr111** [comment]
@Doghousestaffs 
yes, Just testing my code.
Will post an PR soon.

**Comment by bbr111** [comment]
Your UPRN is not working.

I tried with this one 3455011385

We must figure out the type of waste for mapping an icon.

But i don't know these all the types types.

```<BinRound>
<Bin>Empty Bin RES 240 STD</Bin>
<RoundName>Waste-SIERRA 3-110823</RoundName>
<DateTime>11/08/2023 07:00:00</DateTime>
</BinRound>
<BinRound>
<Bin>Empty Bin MIXED REC 55 BOX</Bin>
<RoundName>Waste-DELTA 4-180823</RoundName>
<DateTime>18/08/2023 07:00:00</DateTime>
</BinRound>
<BinRound>
<Bin>Empty Bin MIXED REC 55 BOX</Bin>
<RoundName>Waste-DELTA 4-010923</RoundName>
<DateTime>01/09/2023 07:00:00</DateTime>
</BinRound>
<BinRound>
<Bin>Empty Bin RES 240 STD</Bin>
<RoundName>Waste-SIERRA 3-040823</RoundName>
<DateTime>04/08/2023 07:00:00</DateTime>
</BinRound>
<BinRound>
<Bin>Empty Bin RES 240 STD</Bin>
<RoundName>Waste-SIERRA 3-150923</RoundName>
<DateTime>15/09/2023 07:00:00</DateTime>
</BinRound>```

**Comment by Doghousestaffs** [comment]
3455011390 idk why it’s different just looked up again

**Comment by Doghousestaffs** [comment]
Only thing that’s confusing me with that page is why is it called “missed bin collection”

**Comment by Doghousestaffs** [comment]
Do you have WhatsApp or discord ? @bbr111

**Comment by bbr111** [comment]
@Doghousestaffs 
Yes, but how to share this information non public?

**Comment by Doghousestaffs** [comment]
Which one ?
________________________________

**Comment by bbr111** [comment]
@Doghousestaffs 
I have both. Discord could be better.
But i don't want to share this contact infos here in public.

**Comment by Doghousestaffs** [comment]
That’s fine add me on ravee23
________________________________

**Comment by dt215git** [comment]
**@bbr111**: Looking at a few random addresses, I'd suggest:

RES = Residual = grey bin -> mdi:trash-can
ORG = Organic = brown bin -> mdi:leaf
REC = Recycling = blue bin -> mdi:recycle
MIXED REC = Recycling = blue bin -> mdi:recycle

**Comment by Doghousestaffs** [comment]
Think your probably right tbh
________________________________

**Comment by bbr111** [comment]
resolved by #1161

**Comment by 5ila5** [comment]
added by #1161 
Will be available with release 1.42.0 or is available now as master/git version (up to date clone of this repository)
